framework/requirements/views/web/index.php

```diff
@@ -7,7 +7,7 @@
 <head>
     <meta charset="utf-8"/>
     <title>Yii Application Requirement Checker</title>
-    <?php $this->renderViewFile(dirname(__FILE__) . '/css.php'); ?>
+    <?php $this->renderViewFile(__DIR__ . '/css.php'); ?>
```

Comment:
Should not be modified like that because requirements checker requirements are lower than framework's. It should be able to say that requirements aren't met.